Makefile

```diff
@@ -56,3 +57,13 @@ clean:
 	rm -rf .pytest_cache
 	find . -type d -name __pycache__ -exec rm -rf {} +
 	find . -type d -name .mypy_cache -exec rm -rf {} +
+
+install:
+	yarn install
+	poetry install
+ifeq ($(ARCH),arm64)
+	$(eval ATOMICS := $(shell poetry export | egrep 'atomics==' | awk '{print $$1}'))
+	$(eval VENV := $(shell poetry run poetry env info -p))
```

Comment:
I think you can omit one `poetry` invocation:

```
shell poetry env info -p
```
should be sufficient.

Reply:
wasn't sufficient on my system.
was really struggling on that part and doubling poetry did the trick.